.gitignore

```diff
@@ -25,6 +25,7 @@ wrapper/
 
 # local scripting
 Makefile
+workspace
```

Comment:
nit: what does `workspace` refer to?

Reply:
For ad hoc testing scripts. Scripts run from the context of the monorepo workspace automatically import the packages in the repo, so it's for convenience.

Reply:
we should be able to remove this bit I think, it usually refers to the dir where we do some local testing/experiments